downstream/modules/platform/proc-set-registry-username-password.adoc

```diff
@@ -0,0 +1,23 @@
+[id="proc-set-registry-username-password"]
+
+= Setting registry_username and registry_password
+
+If you intend to use the `registry_username` and `registry_password`` variables in an inventory file you are recommended to use the following method to create a Registry Service Account to set a token with an expiration in the plaintext `inventory/vars.yml` file instead of using a plaintext username and password, for reasons of security.
+
+Registry service accounts provide named tokens that can be used in environments where credentials are shared, such as deployment systems.
+
+.Procedure
+. Navigate to https://access.redhat.com/terms-based-registry/accounts
+. On the *Registry Service Accounts* page click btn:[New Service Account].
+. Enter a name for the account using only the accepted characters
+. Optionally enter a description for the account.
+. Click btn:[Create account].
+. Find the created account in the list. 
+The list of accounts is long so you might have to click btn:[Next] multiple times before finding the account you created. 
+Alternatively, if you know the name of your token, you can go directly to the page by entering the URL https://access.redhat.com/terms-based-registry/token/<name-of-your-token>
```

Comment:
```suggestion
Alternatively, if you know the name of your token, you can go directly to the page by entering the URL \https://access.redhat.com/terms-based-registry/token/<name-of-your-token>
```
This URL isn't a proper link, so maybe make it plain text (the `\` escapes the auto-linking)